Definitions/Chord

```diff
@@ -0,0 +1,29 @@
+class Chord:
+    def __init__(self, symbol, degrees, next, inversion):
```

Comment:
The input would be better as an entire chord definition rather than all the fields broken out into their own arguments. Would look something like:

def __initi__(self, definition):
    self.symbol = definition.get("symbol")
    ...